packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -86,6 +92,32 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [onMouseUp],
   );
 
+  const handleFocus = React.useCallback(
+    (ev: FocusEvent) => {
+      ev.preventDefault();
+      ev.stopPropagation();
+
+      const document = iframeRef.current?.contentDocument;
+      const fiberNav = FiberNavigator.fromDOMNode(ev.target);
+      const parent = fiberNavFindOwnerInJSONTree(fiberNav, jsonTree);
+
+      document?.body.focus();
+
+      // when the selected element was not created by the user, the focus is in the body
+      // therefore we need to warn the user because, in this case, the reader can read the entire document
+      if (document && !parent?.props?.['data-builder-id']) {
+        document.body.style.outline = '4px dashed red';
+        document.body.style.outlineOffset = '-4px';
+        onMessage('Be careful: focus on body!');
```

Comment:
I would propose following message: Warning: Focus on body. Developer might need to handle it and focus relevant element instead.

Reply:
Perfect! I didn't know which message to put, so I've added any so we could discuss on PR. So I will update it. Thx again!